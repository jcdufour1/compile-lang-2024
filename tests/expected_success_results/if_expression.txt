yes
yes2
yes2.2
yes2.20
yes2.30
yes2.30.2
yes2.31
yes2.31.2
yes3
yes3.2
yes4.2
yes4
yes5
yes6.2
yes6
yes7
yes8
