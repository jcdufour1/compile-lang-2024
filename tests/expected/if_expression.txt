yes
yes2
yes3
yes3.2
yes4
yes5
